packages/babel-highlight/test/index.js

```diff
@@ -93,23 +93,5 @@ describe("@babel/highlight", function() {
         });
       });
     });
-
-    describe("when colors are supported", function() {
```

Comment:
Maybe we should test `when colors are not supported` instead?

Reply:
Maybe so @nicolo-ribaudo, currently I'm just learning the code and going through it so I can't say what should be done at this point 😅

Reply:
Yes, I think that it was duplicated because one of them was meant to test when colors were not supported

Reply:
Alright. I will update the pull request with that change.

Reply:
I've updated the pull request 👍🏼